_includes/seo.html

```diff
@@ -4,14 +4,14 @@
 {%- endif -%}
 {%- assign seo_url = seo_url | default: site.github.url -%}
 
-{% assign title_separator = site.title_separator | default: '-' | replace: '|', '&#124;' %}
+{% assign title_separator = site.title_separator | default: '-' %}
```

Comment:
I'd rather keep this line. Liquid `escape_once` doesn't cover the vertical bar. （[Reference](https://www.rubydoc.info/gems/liquid/4.0.0/Liquid/StandardFilters#HTML_ESCAPE_ONCE_REGEXP-constant))